src/commands/game_commands/listen.ts

```diff
@@ -133,9 +146,29 @@ export default class ListenCommand implements BaseCommand {
         }
     };
 
-    call = async ({ message, channel }: CommandArgs): Promise<void> => {
-        const messageContext = MessageContext.fromMessage(message);
-        const guildID = message.guildID;
+    call = async ({ message }: CommandArgs): Promise<void> => {
+        await ListenCommand.startListening(MessageContext.fromMessage(message));
+    };
+
+    static startListening = async (
+        messageContext: MessageContext,
+        interaction?: Eris.CommandInteraction
+    ): Promise<void> => {
+        if (interaction) {
+            await tryCreateInteractionSuccessAcknowledgement(
```

Comment:
We should pass the interaction as a new argument to `sendBeginListeningSessionMessage` and use that as the interaction acknowledgement